config/nginx.conf

```diff
@@ -2,9 +2,24 @@ server {
     listen       8080;
```

Comment:
> this will avoid people being on different versions, unless they are super unlucky and different participants actually loaded the page before & after a deploy

This seems inevitable. If different versions aren't compatible, we should have a way to warn that you joined a room with a host with a newer/older version which may cause issues. Maybe a little warning icon in the corner with the tooltip explaining the version mismatch of any participants.

Or perhaps a warning if Element Call sees signaling in the Matrix room that it doesn't understand. I don't know what kind of incompatibility there was between the versions in question.